stack.yaml

```diff
@@ -4,3 +4,7 @@ packages:
   - co-log
   - co-log-core
   - co-log-benchmark
+
+extra-deps:
+  - chronos-1.0.4
+  - torsor-0.1
```

Comment:
Addition of these dependencies have one problem for `co-log`: it's not possible to add `co-log` to Stackage while these two packages are not there :disappointed: 

I've already opened issue for `torsor`:

* https://github.com/andrewthad/torsor/issues/1

I guess the same should be done for `chronos` as well.

Reply:
I just opened the same issue for `chronos`
- https://github.com/andrewthad/chronos/issues/27

Reply:
https://github.com/commercialhaskell/stackage/pull/4296

Reply:
Nice! I see that it's already on Stackage. So you can bump up resolver version in STACK_YAML:

* https://www.stackage.org/nightly-2019-01-14

Also, since it's a breaking change, we need to bump up `co-log` version from `0.2.0` to `0.3.0` and add entry to changelog about this change.